src/test/java/de/uhd/ifi/se/decision/management/jira/persistence/decisiongroupmanager/TestInsertGroup.java

```diff
@@ -0,0 +1,64 @@
+package de.uhd.ifi.se.decision.management.jira.persistence.decisiongroupmanager;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import de.uhd.ifi.se.decision.management.jira.TestSetUp;
+import de.uhd.ifi.se.decision.management.jira.model.DocumentationLocation;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeElement;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeStatus;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeType;
+import de.uhd.ifi.se.decision.management.jira.model.impl.KnowledgeElementImpl;
+import de.uhd.ifi.se.decision.management.jira.persistence.DecisionGroupManager;
+
+/**
+ * Test class for the persistence of the assigned decision groups.
+ */
+public class TestInsertGroup extends TestSetUp {
+
+    private long id;
+    private String summary;
+    private String description;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'description' could be replaced by a local variable.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5160185)